### ISSUE TITLE
Is there any need to train the model when you only want to check the bounding box regression loss effects?

### ISSUE THREAD

**Issue by dikshaaswal** [david8862/keras-YOLOv3-model-set#88]
Is there any need to train the model when you only want to check the bounding box regression loss effects? 
Or you can just use the trained model?

**Comment by david8862** [comment]
@dikshaaswal  Not quite sure about your requirement. If you just want to compare the bbox prediction outputs with GT on some data samples, only forward propagation on images and collecting regression loss value is enough.

**Comment by dikshaaswal** [comment]
@david8862 I want to compare different diou and giou bounding box regression loss function

**Comment by david8862** [comment]
@dikshaaswal then I think training model with different loss should be mandatory

**Comment by dikshaaswal** [comment]
Thanks :)
